src/SFA.DAS.EmployerIncentives.Commands/ApprenticeshipIncentive/RefreshLearner/RefreshLearnerCommandHandler.cs

```diff
@@ -47,6 +55,13 @@ public async Task Handle(RefreshLearnerCommand command, CancellationToken cancel
 
             learner.SetLearningPeriods(learnerData.LearningPeriods(incentive));
 
+
+            if (learner.SubmissionData != null && !learner.SubmissionData.LearningFoundStatus.LearningFound)
+            {
+                _logger.LogInformation("Matching ILR record not found for ApprenticeshipIncentiveId: {ApprenticeshipIncentiveId}, ApprenticeshipId: {ApprenticeshipId}, UKPRN: {UKPRN}, ULN: {ULN} with reason: {NotFoundReason}",
+                    learner.ApprenticeshipIncentiveId, learner.ApprenticeshipId, learner.Ukprn, learner.UniqueLearnerNumber, learner.SubmissionData.LearningFoundStatus.NotFoundReason);
```

Comment:
This is the only one required to meet the requirement.

Reply:
Additional logging added as per request from test team and personal experience while supporting and debugging issues in TEST